syncopy/specest/compRoutines.py

```diff
@@ -1,18 +1,29 @@
 # -*- coding: utf-8 -*-
 #
 # Definition of the respective ComputationalRoutines
-# for the `freqanalysis` frontend.
+# for the `freqanalysis` frontend. The *_cf function
+# definitions serve as middleware connecting the pure
+# backend methods to the ComputationalRoutines. The
+# 1st argument always is the data, and the last argument
+# `method_kwargs` gets passed as is to the underlying
+# backend, so that the following calling signature
+# is valid for all backend methods: backend_method(data, **method_kwargs)
```

Comment:
This is great!

Reply:
If I'm not mistaken, only wavelet and superlet methods use `method_kwargs` atm, right?

Reply:
Yap.. so I am working on getting it also in this shape for the remaining 2 specest methods!